app/scripts/controllers/reports/aboRhGroupsReport.js

```diff
@@ -138,13 +138,19 @@ angular.module('bsis')
       paginationPageSizes: [10],
       paginationTemplate: 'views/template/pagination.html',
       columnDefs: columnDefs,
+      
+      exporterPdfOrientation: 'landscape',
+      exporterPdfPageSize: 'A4',
+      exporterPdfDefaultStyle: {fontSize: 8, margin: [-2, 0, 0, 0] },
+      exporterPdfTableHeaderStyle: {fontSize: 8, bold: true, margin: [-2, 0, 0, 0] },
+      exporterPdfMaxGridWidth: 650,
```

Comment:
I believe it needs to be 550 in order to fit on the page landscape. I just printed out the PDF and the table is cut off on the right. I worked on the Test Batch report and noticed that the ui-grid documentation was incorrect.